nestjs/src/courses/dto/used-course.dto.ts

```diff
@@ -0,0 +1,16 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsString } from 'class-validator';
+
+export class UsedCourseDto {
+  constructor(obj: { name: string; isActive: boolean }) {
```

Comment:
Maybe it's better to use a more understandable naming, like 'course' instead of 'obj'?

Reply:
You're right. Fixed